src/rebar3_hank_prv.erl

```diff
@@ -115,11 +115,21 @@ get_rules(State) ->
     end.
 
 normalize(IgnoreRules) ->
-    lists:map(fun ({Wildcard, Rule, Options}) ->
-                      {Wildcard, Rule, Options};
-                  ({Wildcard, Rule}) ->
-                      {Wildcard, Rule, all};
-                  (Wildcard) ->
-                      {Wildcard, all, all}
-              end,
-              IgnoreRules).
+    lists:foldl(fun ({_, _, _} = WildcardRuleOpts, Acc) ->
+                        normalize_rules(WildcardRuleOpts, Acc);
+                    ({_, _} = WildcardRule, Acc) ->
+                        normalize_rules(WildcardRule, Acc);
+                    (Wildcard, Acc) ->
+                        [{Wildcard, all, all} | Acc]
+                end,
+                [],
+                IgnoreRules).
```

Comment:
```suggestion
lists:foldl(fun (Wildcard, Acc) when is_tuple(Wildcard) ->
                        normalize_rules(Wildcard, Acc);
                    (Wildcard, Acc) ->
                        [{Wildcard, all, all} | Acc]
                end,
                [],
                IgnoreRules).
```

Just a suggestion to avoid both calls. _In cases of bad tuples (different than 2 or 3 elements) it will crash in `normalize_rules/2`_

Reply:
Sure, but wouldn't it already otherwise? 🤣

Reply:
(I think your proposed change will make CI throw, because of `format`)

Reply:
Oh, wait, I mis-read your comment. You're trying to trim code here. Makes sense. I'll get on it.

Reply:
Done: decaa56